common/src/main/scala/monix/connect/common/Operators.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2014-2020 by The Monix Project Developers.
+ * See the project homepage at: https://monix.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package monix.connect.common
+
+import monix.reactive.Observable
+
+object Operators {
+
+  type Transformer[A, B] = Observable[A] => Observable[B]
+
+  object Implicits {
+    implicit class ObservableExtension[A](ob: Observable[A]) {
+      def transform[B](transformer: Transformer[A, B]): Observable[B] = {
```

Comment:
Hmm, maybe we should have kept this in Monix 🙂

Reply:
There is some demand: https://github.com/monix/monix/issues/891